### DISCUSSION TITLE
Search Result pages doesn`t display proper image of news post

### DISCUSSION THREAD

**Discussion by ajitsingh370** [microsoft-search/pnp-modern-search#3234] [Ideas]
Hi Team,
We have issue with Modern pnp Search Result web part V4
We are trying to display news pages from different site. The pages are displayed but thesome images are not displaying. (Please find the attached screenshot). This problem comes only to one specific site. Week ago it was working properly. It usually takes the header image of the page, but for some reason it stop working.

I have seen the same issue with case no. 1604 is close. But I am still facing this issue. Is there any timeline to resolve this issue?

https://github.com/microsoft-search/pnp-modern-search/issues/1604

To Reproduce

Add a search results in a page.
Create a search query with PromotedState2 and Content type News Page
Create a News Page in the same site
Add a header image of the page and publish it
In the Search Result web part The image doesn`t appear. (Pl
Expected behavior
The Header image of the News page should appear in Search Results Boxes.

Screenshots
![PNPSearch webpart Issue2](https://github.com/microsoft-search/pnp-modern-search/assets/144081939/12af8257-9f15-4183-a39d-b29ba8773894)

**Comment by kasperbolarsen** [comment]
Hey :-)
please have a look at https://microsoft-search.github.io/pnp-modern-search/using-query-tools-to-verify-issues/ for tips and tricks. Using the SP Query Tool I could see that the image you are looking for seems to be in the PictureThumbnailURL property. Update the Layout slots in the Results Web Part to use this property and the image should appear :-)
